src/data/firestore/sections.ts

```
@@ -0,0 +1,78 @@
+import { db } from "@/config/firebase";
+import { Section, SectionID, CommonSection, CommonSectionID, SchedulingSection, SchedulingSectionID } from "@/types/sessionTypes";
```

ESLint

[error] 'CommonSection' is defined but never used.

(@typescript-eslint/no-unused-vars)